### ISSUE TITLE
Microsoft Docs cross reference service cannot find all content (Member ID URL not found for target)

### ISSUE THREAD

**Issue by apgrucza** [EWSoftware/SHFB#744]
The change in [SHFB version 2019.8.24.0](http://ewsoftware.github.io/SHFB/html/3bb3e240-7c36-456b-bebd-972ff1508c15.htm) to use the Microsoft Docs cross reference service instead of the MTPS content service has caused build warnings in my project, such as the ones below.

```
BuildAssembler : warning : ResolveReferenceLinksComponent: [****] Member ID URL not found for target 'T:Microsoft.Expression.Interactivity.Core.ChangePropertyAction'.
BuildAssembler : warning : ResolveReferenceLinksComponent: [****] Member ID URL not found for target 'T:System.Windows.Interactivity.InvokeCommandAction'.
```

For example, in previous versions of SHFB, `T:Microsoft.Expression.Interactivity.Core.ChangePropertyAction` would link to http://msdn2.microsoft.com/en-us/library/ff726452, which redirects to https://docs.microsoft.com/en-us/previous-versions/visualstudio/design-tools/expression-studio-4/ff726452(v=expression.40)?redirectedfrom=MSDN. This page has a "We're no longer updating this content regularly" warning so this might be why it's not available through the Microsoft Docs cross reference service, even though the content does exist at docs.microsoft.com (under "Previous Versions").

I'm not sure what the best solution is but one option might be to make SHFB (optionally?) fallback to the MTPS content service if the content cannot be found via the Microsoft Docs cross reference service.

**Comment by EWSoftware** [comment]
The MTPS content service is obsolete and may go away at any time.  It's been rather unreliable of late which is why it was replaced.  The long term plan is to create a build component that lets you map the missing IDs to a URL in some fashion.  I don't have a time frame for when that might be done though.

**Comment by EWSoftware** [comment]
Merged with #899
